mysql-test/suite/binlog_encryption/t/binlog_encryption_without_keyring.test

```diff
@@ -0,0 +1,48 @@
+# Bug: PS-288: mysqld got signal 6 if you try to enable encrypt_binlog + binary log without keyring-file plugin
+
+# Server is aborted when binlog_error_action is set to ABORT_SERVER and binlog encryption could not be
+# initialized. An error message generated in server log got improved in this case by mentioning that
+# encryption could be a reason why server got aborted.
+
+--source include/not_embedded.inc
+--source include/have_log_bin.inc
+
+--let restart_hide_args=1
+--let $restart_parameters=restart:--log-error=$MYSQLTEST_VARDIR/tmp/binlog_encryption_without_keyring.err
+--source include/restart_mysqld.inc
+
+UNINSTALL PLUGIN keyring_file;
+
+--echo # Make sure that no keyring plugin is installed
+query_vertical SELECT PLUGIN_NAME,PLUGIN_VERSION,PLUGIN_STATUS
+FROM INFORMATION_SCHEMA.PLUGINS WHERE plugin_name LIKE 'keyring%';
+
+--let $assert_text= Binlog should be ON
+--let $assert_cond= "[SELECT @@GLOBAL.log_bin = 1]" = 1
+--source include/assert.inc
+
+--let $assert_text= Binlog encryption should be ON
+--let $assert_cond= "[SELECT @@GLOBAL.encrypt_binlog = 1]" = 1
+--source include/assert.inc
+
+--let $saved_binlog_error_action=`SELECT @@GLOBAL.binlog_error_action`
```

Comment:
Can you combine this restart with the previous one?

Reply:
This is not exactly restart - I am going make the test to abort the server, because RESET MASTER cannot encrypt binlog - no keyring plugin installed. I am restarting at the beginning as you suggested with custom log file - which I need to check for pattern when RESET MASTER will cause server to abort.